### DISCUSSION TITLE
유니티에 프리펩화 오브젝트의 인스펙터 및 마테리얼 질문

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#409] [5. Q&A]
인스펙터로 지정할수 있는 시작지점과 끝 지점을 가지고 있는, 이동하는 오브젝트의 프리펩이 있습니다

해당 오브젝트의 시작지점과 끝 지점을 인스펙터에 넣고 실행시키면, 인스펙터 내부 값이 기존 프리펩에 저장된 값으로 초기화됩니다.

즉, 시작지점과 끝지점에 0,0,0으로 넣고 프리펩으로 만들었다면 해당 오브젝트를 다른곳에 배치해서 다른 값을 넣어도, 0,0,0이 됩니다.

프리펩화 하기 전에는 값을 변경하면 변경된대로 잘 되었는데, 어떻게 해야 할까요? 프리펩을 사용하지 말까요?
![질문1](https://user-images.githubusercontent.com/73985764/154874661-9689d30f-6158-4578-9713-66dbf6f07340.JPG)
프리펩화 한 오브젝트. 인스펙터에 값을 변경하여 넣음.
![질문2](https://user-images.githubusercontent.com/73985764/154874699-ea912112-8302-43e7-bb64-647ffbd2c09c.JPG)
테스트 플레이 영상. 인스펙터 값이 기본 프리펩값인 start 0,0,0 endpos 0,0,0으로 초기화됨.

또한 오브젝트의 메쉬필터와 매쉬 랜더러 내 마테리얼이 프리펩화 하면 삭제되는 경우도 있습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 영상이 첨부되지 않은 것 같습니다.🥲
말씀주신 이슈는 게임오브젝트를 프리팹으로 만들면 위치 값이 초기화되거나, 
메쉬필터 또는 메쉬 랜더러가 삭제되는 현상이 나타난다고 이해하면 될까요?

**Reply by ted002** [reply]
게임오브젝트를 프리팹화 하면, 인스펙터 내에 있는 제페토 스크립트의 값이 초기화 됩니다.
https://youtu.be/7ARUGVwXMGs 영상링크와 같이요.

**Reply by Zepeto-TechEvangelist** [reply]
상세한 영상 설명 감사합니다.
내부 확인 후 말씀 드리겠습니다~!

**Comment by Andwhy81kr** [comment]
https://youtu.be/7ARUGVwXMGs
이문제군요.

프리팹으로 만들어진 객체는 인스팩터에서 변경해도 초기화 되고 있습니다.
저는 1.1.3 버전만 확인했는데, 다른버전은 잘모르겠네요.

이것말고도 다른 버그들도 좀 찾는데로 올리도록하겠습니다.

**Reply by ted002** [reply]
네 맞습니다 이런느낌입니다

**Reply by Zepeto-TechEvangelist** [reply]
1.1.5에서도 재현되고 있어서 내부 확인 후 수정 하도록 하겠습니다.
두 분 제보 감사합니다.
